cypress/e2e/what-happens-next.cy.js

```diff
@@ -0,0 +1,14 @@
+describe('What happens next', () => {
+    it('has a continue button', () => {
+        cy.visit('/testing-start?redirect=/what-happens-next');
+
+        cy.injectAxe();
+        cy.checkA11y(null, {
+            rules: { region: { enabled: false } },
+        });
+
+        cy.contains('a', 'Continue').click();
+
+        cy.url().should('contain', '/what-happens-when-signing');
```

Comment:
these tests are weak, once the whole flow is in we should probably replace with one that just clicks through all the signing content